AdyenActions/Components/Await/PollingComponent.swift

```diff
@@ -82,7 +85,8 @@ internal final class PollingComponent: AnyPollingHandler {
         case let .success(response):
             return shouldRetry(response, paymentData: paymentData)
         case .failure:
-            return false
+            errorCount += 1
+            return errorCount < maxErrorNumber
```

Comment:
you should reset the counter in `startPolling` and `didCancel`, also change the unit tests.

Reply:
On `startPolling` should be enough, right ?

Reply:
what's wrong with doing it in `didCancel` as well?

Reply:
On `startPolling` would be sufficient.
Canceled or finalised, It is not going to start on it's own, right ? :)